### ISSUE TITLE
[FEATURE] Custom rates profiles indepedent of quad profile

### ISSUE THREAD

**Issue by meekaah** [FlightOne/FalcoX-Issue-Tracker#93]
**Is your feature request related to a problem? Please describe.**
I cant change rate profile and then come back to MY custom usual rates easily to test out new rates

**Describe the solution you'd like**
I'd love to be able to save MY custom rates as a profile, indepedent of quad profiles
so I can keep the pids same etc but try out new rates easily.

**Describe alternatives you've considered**
I THINK as a workaround you can use quad profiles but if you make one change on the pids or whatever you have to edit all your other profiles just to have different rates 

**Additional context**
If there's another solution i'm not aware i'd love to know. thanks

**Comment by felipeng** [comment]
yes!! This is the biggest annoying stuff I've noticed since I started using FalcoX. Also a friend of mine also complains about it. I use the workaround using profiles, but doesn't work nicely as you stated.
